app/views/layouts/application.html.erb

```diff
@@ -31,28 +33,22 @@
   <body class="govuk-template__body">
     <%= javascript_include_tag 'js_check' %>
 
-    <% if ENV["GOOGLE_ANALYTICS_ID"] %>
-      <noscript>
-        <iframe
-          src="https://www.googletagmanager.com/ns.html?<%= ENV["GTM_ANALYTICS"] %>"
-          height="0"
-          width="0"
-          style="display:none;visibility:hidden">
-        </iframe>
-      </noscript>
+    <% if cookies_accepted? %>
+      <% if ENV["GOOGLE_ANALYTICS_ID"] %>
+        <noscript>
+          <iframe
+            src="https://www.googletagmanager.com/ns.html?<%= ENV["GTM_ANALYTICS"] %>"
+            height="0"
+            width="0"
+            style="display:none;visibility:hidden">
+          </iframe>
+        </noscript>
+      <% end %>
     <% end %>
 
-    <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
+    <%= render "shared/cookie_banner" %>
 
-    <div id="global-cookie-message" class="govuk-cookie-banner" role="region" aria-label="cookie banner">
-      <div class="govuk-width-container">
-        <p class="govuk-cookie-banner__message"><%= journey_service_name(current_journey_routing_name) %> uses cookies to make the site simpler.</p>
-        <div class="govuk-cookie-banner__buttons">
-          <button id="accept-cookies" class="govuk-button" type="submit" data-module="govuk-button">Accept cookies</button>
-          <%= link_to('Cookie information', cookies_path(current_journey_routing_name), class: "govuk-cookie-banner__link" ) %>
-        </div>
-      </div>
-    </div>
+    <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
```

Comment:
documentation [1] dictates cookies banner should appear before the main content

[1] https://design-system.service.gov.uk/components/cookie-banner/#how-it-works